microsoft-edge/privacy-whitepaper/index.md

```diff
@@ -534,6 +534,36 @@ To enhance the Kids Mode experience, Microsoft Edge adds a safe keyword to the h
 Kids Mode does not collect data for personalization of the news feed or other Microsoft services.  You may not change the privacy settings for Kids Mode.  Other settings like Windows Defender SmartScreen and diagnostic data are configured according to the profile in which Kids Mode was launched.  For more information about diagnostic data about how you use the browser and Windows Defender SmartScreen, see the [Diagnostic Data](#diagnostic-data) and [SmartScreen](#smartscreen) sections.
 
 
+<!-- ====================================================================== -->
+## Magnify Image<!-- todo: is the menuitem capital I or lowercase?  global in section -->
+
+To use the Magnify Image feature, right-click an image and then select **Magnify Image**, or press the **Ctrl** key twice while hovering over an image.  The Magnify Image feature retrieves the largest available version of the image from the website and displays it in a popup window.
+
+
+<!-- ------------------------------ -->
+#### AI-based image enhancement
+
+If your browser has WebNN enabled, there's also a button<!-- todo: what's the button label?  can we commit to specifying the button label at this time?  [click] the **Foo** button --> to use AI to enhance the image.  To check whether WebNN is enabled, go to the `edge://gpu` page, and then in the **Graphics Feature Status** section, see whether **WebNN** says **Enabled**:<!-- todo: how to control whether WebNN is enabled? -->
+
+![The status of the WebNN setting, shown in the edge://gpu page](./index-images/webnn-disabled-gpu.png)
+
+AI-based image enhancement is done on your device by using a model that's downloaded from Microsoft servers when you click the AI enhancement<!-- todo: what's the button label?  can we commit to specifying the button label at this time?  [click] the **Foo** button --> button.  Neither the image nor any personally identifiable information is sent to Microsoft.
+
+
+<!-- ------------------------------ -->
+#### Turning off the feature
+
+To turn off the Magnify Image feature, use the command-line flag `msEdgeImageMagnifyUI`.
+<!-- todo: state the gui nav path, if any: eg: 
```

Comment:
is there a gui nav path instead of having to use command-line flag?  this article is for end users, not devs.
@sjdallst2 

rendered article section:
https://review.learn.microsoft.com/microsoft-edge/privacy-whitepaper/?branch=pr-en-us-3234#magnify-image

Reply:
.md file doesn't contain "command line" or "command-line".  

this article (for users) is becoming more separate - elsewhere in the Edge Dev docset (for devs) -- mentions Edge command-line options.
command line: 24 hits in 18 files
command-line: 43 hits in 20 files
eg the wording in: 
https://learn.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/whats-new/2023/09/devtools-117#new-experiment-explain-console-errorswarnings-with-copilot-in-sidebar

That example shows two ways: edge://flags flag, & start Edge w/ command-line option.  That example has clear specific wording to possibly use here.

Reply:
result:
```
To turn off the Magnify Image feature, use 
the command-line flag `msEdgeImageMagnifyUI`.  
For example, run Microsoft Edge from the 
command line using the `enable-features` 
command-line option, as follows:

msedge.exe --enable-features=msEdgeImageMagnifyUI
```

Reply:
Thanks for finding out how to write this out. The only issue is that to turn off the feature you would use --disable-features.

To turn off the Magnify Image feature, use 
the command-line flag `msEdgeImageMagnifyUI`.  
For example, run Microsoft Edge from the 
command line using the `disable-features` 
command-line option, as follows:

msedge.exe --disable-features=msEdgeImageMagnifyUI